war/src/main/java/io/lumeer/engine/controller/DatabaseInitializer.java

```diff
@@ -40,6 +45,11 @@
    @Inject
    private DataStorageDialect dataStorageDialect;
 
+
+   public void init(@Observes @Initialized(RequestScoped.class) Object init) {
```

Comment:
Why is this method empty? Should not the initialization be done here instead of in `@PostConstruct` method? And what about changing `@Initialized` to use `ApplicationScoped` instead?